js/components/NavCard.js

```diff
@@ -8,9 +8,18 @@ import { IconProp, FEATHER } from '../../data/fontFamilies';
 
 import Card from './Card';
 
-export default function NavCard({ title, description, onPress, icon }) {
+export default function NavCard({
+  title,
+  description,
+  onPress,
+  icon,
+  smallMode,
+}) {
   return (
-    <Card onPress={onPress} style={{ padding: 24 }}>
+    <Card
+      onPress={onPress}
+      style={[{ padding: 24 }, smallMode && { padding: 10, paddingLeft: 16 }]}
```

Comment:
```suggestion
      style={smallMode ? { padding: 10, paddingLeft: 16 } : { padding: 24 }}
```
since these are mutually exclusive

Reply:
Done!